packages/components/RadioGroup/src/RadioButton.tsx

```diff
@@ -26,7 +26,7 @@ export const RadioButton = compose<IRadioButtonType>({
 
     /* We don't want to call the user's onChange multiple times on the same selection. */
     const changeSelection = () => {
-      if (buttonKey != info.selectedKey) {
+      if (buttonKey != info.selectedKey && !disabled) {
```

Comment:
Just curious, in what scenario ill changeSelection() be called when the button is disabled? Doesn't disabling the button disallow focus/press on that button?

Reply:
I think this has to do with removing the filterViewProps from button. The button is selectable even when it's disabled...